File: app/src/main/java/com/example/project_note/MainActivity.java
package com.example.project_note;


import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;




public class MainActivity extends AppCompatActivity{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        // Hide the status bar
        View decorView = getWindow().getDecorView();
        int uiOptions = View.SYSTEM_UI_FLAG_FULLSCREEN;
        decorView.setSystemUiVisibility(uiOptions);



        openHomeFragment();
    }

    private void openHomeFragment() {
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.frag_layout,new HomeFragment());
        fragmentTransaction.commit();

    }


}

File: app/src/main/java/com/example/project_note/MainActivity.kt
package com.example.project_note

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.ViewModelProvider
import com.example.project_note.ViewModal.ViewModalNote
import com.example.project_note.fragment.HomeFragment

class MainActivity : AppCompatActivity() {
    private lateinit var viewModel: ViewModalNote

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        viewModel = ViewModelProvider(this).get(ViewModalNote::class.java)

        if (savedInstanceState == null) {
            // Hiển thị FragmentA khi lần đầu tiên mở ứng dụng
            openHomeFragment()
        }

    }
        fun openHomeFragment() {
            supportFragmentManager.beginTransaction()
                .replace(R.id.frag_layout, HomeFragment(), "Home")
                .addToBackStack(null)
                .commit()
        }

    }
